packages/boot/tools/supports.ts

```diff
@@ -294,6 +294,8 @@ export const AckBehavior = {
   Queued: 'QUEUED',
   /** inbound messages are delivered immediately */
   Immediate: 'IMMEDIATE',
+  /** inbound responses never arrive (to simulate mis-configured connections etc.) */
+  Never: 'NEVER',
```

Comment:
nice. do we need this implemented in all cases?

Reply:
there are other cases? help?